### PR TITLE
update docs_zh-CN/changelog.md

### PR DESCRIPTION
**Pull request by kai422**
Add Chinese translation of docs/changelog.md

### CODE REVIEW

File: docs_zh-CN/changelog.md [hunk 1]
```diff
@@ -1 +1,246 @@
...
+## v0.10.0 (12/08/2021).
+
+**亮点**
+
+1. 支持 LIIF-RDN (CVPR'2021)
+2. 支持 BasicVSR++ (NTIRE'2021)
+
+**新功能**
+
+- 支持从文件加载对 video SR datasets 的标注 ([#423](https://github.com/open-mmlab/mmediting/pull/423))
```
Comment (innerlee) [inline docs_zh-CN/changelog.md:12]: Video SR datasets 支持加载文件列表

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Thanks for the contribution!

**Comment by innerlee** [comment]
Hi @kai422 The linting fails. Please run `pre-commit install` and `pre-commit run --all-files`. Details see the contribution guide https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md

**Comment by OpenMMLab-Coodinator** [comment]
Hi @kai422 ！First of all, we want to express our gratitude for your significant PR in the MMEditing project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**